main.go

```diff
@@ -117,6 +118,14 @@ func init() {
 	utilruntime.Must(managementv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(obv1.AddToScheme(scheme))
 	utilruntime.Must(routev1.AddToScheme(scheme))
+
+	ns, err := getOperatorNamespace()
+	if err == nil {
+		operatorNamespace = ns
+	} else {
+		operatorNamespace = defaultOperatorNamespace
```

Comment:
The `getOperatorNamespace` function returns the `defaultOperatorNamespace` in case of "_In cluster_" run.
In case of "_Out of the cluster_" run, if the namespace has not been created and we use the `defaultOperatorNamespace` then the get `ConfigMaps` will fail. If it is not possible to find a namespace in the case of _Out of the cluster_ run then the operator should fail.

Reply:
Check now, I think that I fixed :-)